src/js/edu-benefits/1990e/helpers.jsx

```diff
@@ -0,0 +1,10 @@
+import { transformForSubmit } from '../../common/schemaform/helpers';
+
+export function transform(formConfig, form) {
```

Comment:
Yeah, the purpose of this is to transform the data into the body for the POST call. There's a default one, but it won't have the `educationBenefitsClaims` structure that the backend expects.

The review page should generate either way, but we do need at least this code for all the edu forms to POST data correctly. I'm not sure about the non-edu forms we're going to build, so I didn't try to generalize it more.